chapter/8/big-data.md

```diff
@@ -1,11 +1,229 @@
 ---
 layout: page
 title:  "Large Scale Parallel Data Processing"
-by: "Joe Schmoe and Mary Jane"
+by: "Jingjing and Abhilash"
 ---
+## Introduction
+`JJ: Placeholder for introduction` The booming Internet has generated big data...
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. {% cite Uniqueness --file big-data %}
 
-## References
+This chapter is organized in <label for="note1" class="margin-toggle sidenote-number"></label><input type="checkbox" id="note1" class="margin-toggle"/><span class="sidenote">JJ: need to fill in more stuff</span>
+
+- **Data paralleling**:
+  - MapReduce {% cite dean2008mapreduce  --file big-data %}
+  - FlumeJava {% cite chambers2010flumejava --file big-data %}
+  - ...
+- **Graph paralleling**:
+  - Pregel
+  - ...
+
+For each programming model, we will discuss the motivation, basic model, execution model, fault-tolerance and performance.
```

Comment:
I agree on the comments, and we are going to the programming vs. execution direction. We had a discussion today about this. The push was from a while ago and I didn't really have time to update that, sorry for the confusion, and I am working on writing introduction (discussions over models to cover and relationships between them) now, and will address these points.